src/main/java/org/example/springboot_labb2/controller/WebController.java

```diff
@@ -6,8 +6,15 @@
 import org.example.springboot_labb2.service.UserService;
 import org.springframework.stereotype.Controller;
 import org.springframework.ui.Model;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
```

Comment:
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.PathVariable;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RequestParam;

these imports are unecessary because we already have:

import org.springframework.web.bind.annotation.*;

this imports all annotations needed.